File: Bee-Crowd/Exercises_Python/1094.py
linhas = int(input())

lista = []
soma_dos_int = 0
soma_dos_coelhos = 0
soma_dos_ratos = 0
soma_dos_sapos = 0
for count in range(0,linhas):
    entrada = input().split()
    lista.append((int(entrada[0]), str(entrada[1])))
    soma_dos_int = soma_dos_int + int(entrada[0])
    if entrada[1] == 'C':
        soma_dos_coelhos = soma_dos_coelhos + int(entrada[0])
    if entrada[1] == 'R':
        soma_dos_ratos = soma_dos_ratos + int(entrada[0])
    if entrada[1] == 'S':
        soma_dos_sapos = soma_dos_sapos + int(entrada[0])

percentual_c = (soma_dos_coelhos / soma_dos_int) * 100
percentual_r = (soma_dos_ratos / soma_dos_int) * 100
percentual_s = (soma_dos_sapos / soma_dos_int) * 100

print('Total:',soma_dos_int,'cobaias')
print('Total de coelhos:',soma_dos_coelhos)
print('Total de ratos:',soma_dos_ratos)
print('Total de sapos:',soma_dos_sapos)
print('Percentual de coelhos: {:.2f}'.format(percentual_c),'%')
print('Percentual de ratos: {:.2f}'.format(percentual_r),'%')
print('Percentual de sapos: {:.2f}'.format(percentual_s),'%')

File: Bee-Crowd/Exercises_C++/1094.cpp
#include <iostream>
using namespace std;

int main() {

    int qntd_de_vezes;
    cin >> qntd_de_vezes;

    int n,count,soma,coelhos,ratos,sapos;
    double p_coelhos, p_ratos, p_sapos;
    string letra;

    soma = 0;
    coelhos = 0;
    ratos = 0;
    sapos = 0;
    count = 0;
    while (count < qntd_de_vezes) {
        cin >> n >> letra;

        if (letra == "C") {
            coelhos += n;
            soma += n;
        }
        else if (letra == "R") {
            ratos += n;
            soma += n;
        }
        else if (letra == "S") {
            sapos += n;
            soma += n;
        }
        count += 1;
    }

    p_coelhos = 100 * (1.0 * coelhos / soma);
    p_ratos = 100 * (1.0 * ratos / soma);
    p_sapos = 100 * (1.0 * sapos / soma);

    cout << "Total: " << soma << " cobaias" << endl;
    cout << "Total de coelhos: " << coelhos << endl;
    cout << "Total de ratos: " << ratos << endl;
    cout << "Total de sapos: " << sapos << endl;
    printf("Percentual de coelhos: %.2f %%\n",p_coelhos);
    printf("Percentual de ratos: %.2f %%\n",p_ratos);
    printf("Percentual de sapos: %.2f %%\n",p_sapos);
}